client/src/InfoBase/index_data.js

```diff
@@ -95,7 +95,7 @@ const index_lang_lookups = {
   },
   bonus_footer_list: {
     en: `
-     <li><a target="_blank" rel="noopener noreferrer" href="https://www.canada.ca/en/contact.html">Contact us</a></li>
+     <li><a target="_self" rel="noopener noreferrer" href="#contact">Contact us</a></li>
```

Comment:
`_self` is the default `target` value, can drop it. `rel="noopener noreferrer"` is only necessary, for security reasons, on `target="_blank"` links, preferable to remove those attributes from an internal link as well.